app/mbm/static/js/geolocation.js

```diff
@@ -0,0 +1,36 @@
+// Manages geolocation related functionality for the app
+//
+// TODO: store the result of `watchPosition()` and remove the handler when we 
+// don't need the user's location
+export default class Geolocation {
+  constructor(app) {
+    this.app = app
+    this.markerName = 'gpslocation'
+    this.startGPSTracking()
+  }
+
+  startGPSTracking() {
+    if (navigator.geolocation) {
+      navigator.geolocation.watchPosition(this.handleGPSPositionUpdate.bind(this))
```

Comment:
should we be using `enableHighAccuracy` and/or set a `maximumAge` for some caching? https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition#options